front/src/applications/operationalStudies/components/MacroEditor/osrdToNge.ts

```diff
@@ -177,6 +183,13 @@ const castNodeToNge = (
   ),
 });
 
+export const getTrainrunCategories = (t: TFunction<'operational-studies'>): TrainrunCategory[] =>
+  Object.entries(OSRD_TRAINRUN_CATEGORY_MAPPING).map(([key, category]) => ({
```

Comment:
If OSRD_TRAINRUN_CATEGORY_MAPPING is a Map, you can iterate in the values like this:
```suggestion
  Array.from(OSRD_TRAINRUN_CATEGORY_MAPPING.entries()).map(([key, category]) => ({
```

Reply:
Done